File: AppLojaCursos/src/app/DTO/transacao.dto.ts
export class TransacaoDTO {
    ID_Transacao: number;
    ID_Curso: number;
    Valor: number;
    DataTransacao: Date;
    Email_Usuario: string;

    constructor(ID_Curso: number, Valor: number, Email_Usuario: string){
        this.ID_Transacao = null;
        this.ID_Curso = ID_Curso;
        this.Valor = Valor;
        this.Email_Usuario = Email_Usuario;
        this.DataTransacao = new Date(Date.now());
    }
}

File: ApiLojaCursos/DTO/TransacaoDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DTO
{
    public class TransacaoDTO
    {
        public long ID_Transacao { get; set; }
        public long ID_Curso { get; set; }
        public decimal Valor { get; set; }
        public DateTime DataTransacao { get; set; }
        public string Email_Usuario { get; set; }
    }
}
